_includes/navbar.html

```diff
@@ -0,0 +1,31 @@
+<header id="navigation" class="p-navigation">
+    <div class="p-navigation__banner">
+      <div class="p-navigation__logo">
+        <a class="p-navigation__link is-selected" href="{{ site.baseurl }}/">
+          <img class="p-navigation__image" src="{{ site.baseurl }}/assets/img/logomark.svg" alt="alice &bull; os&nbsp;">
+          beta
+        </a>
+      </div>
+      <a href="#navigation" class="p-navigation__toggle--open" title="menu">Menu</a>
+      <a href="#navigation-closed" class="p-navigation__toggle--close" title="close menu">Close menu</a>
+    </div>
+    <nav class="p-navigation__nav" role="menubar">
+      <span class="u-off-screen">
+        <a href="#main-content">Jump to main content</a>
+      </span>
+      <ul class="p-navigation__links" role="menu">
+        <li class="p-navigation__link" role="menuitem">
+                <a href="https://docs.aliceos.app">Docs</a>
+              </li>
+              <li class="p-navigation__link {% if page.url == '/features/' %} is-selected{% endif %}" role="menuitem">
+                <a href="{{ site.baseurl }}/features/">Features</a>
+              </li>
+        <li class="p-navigation__link" role="menuitem">
+          <a href="https://github.com/TheAngelReturns/aliceos">GitHub</a>
+        </li>
+        <li class="p-navigation__link {% if page.url == '/studio/' %} is-selected{% endif %}" role="menuitem">
+            <a href="{{ site.baseurl }}/studio/">The Studio</a>
```

Comment:
you do not need ``{{site.baseurl}}`` on this. Just use `/` As per Jekyll will do that for you